packages/rancher-istio/charts/samples/overlay-example.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  components:
+    ingressGateways:
+    - enabled: true
+      name: istio-ingressgateway
+    - enabled: true
+      k8s:
+        resources:
+          requests:
+            cpu: 200m
+        service:
+          ports:
+          - name: tcp-citadel-grpc-tls
+            port: 8060
+            targetPort: 8060
+          - name: tcp-dns
+            port: 5353
+        serviceAnnotations:
+          cloud.google.com/load-balancer-type: internal
+      name: ilb-gateway
+      namespace: user-ingressgateway-ns
+    - enabled: true
+      k8s:
+        resources:
+          requests:
+            cpu: 200m
+        service:
+          ports:
+          - name: tcp-citadel-grpc-tls
+            port: 8060
+            targetPort: 8060
+          - name: tcp-dns
+            port: 5353
+        serviceAnnotations:
+          cloud.google.com/load-balancer-type: internal
```

Comment:
two is probably enough, the 3rd one looks the same ?

Reply:
I will remove the first one and the leave the two that look the same (they are, but the namespace is different and I want to highlight)